lib/path-reservations.js

```diff
@@ -116,8 +117,7 @@ module.exports = () => {
     // So, we just pretend that every path matches every other path here,
     // effectively removing all parallelization on windows.
     paths = isWindows ? ['win32 parallelization disabled'] : paths.map(p => {
-      return stripSlashes(normPath(join(p)))
-        .normalize('NFKD')
+      return normalize(stripSlashes(normPath(join(p))))
```

Comment:
I know stripSlashes and normPath aren't specifically the culprits here but if we're caching, do we get any meaningful improvements by doing them inside the shared normalize function?